ReactWindows/ReactNative/UIManager/PropertySetter.cs

```diff
@@ -114,8 +115,15 @@ protected object ExtractProperty(ReactStylesDiffMap props)
                 return defaultFunc(_attribute);
             }
 
-            return props.GetProperty(Name)?
-                .ToObject(PropertyType);
+            var prop = props.GetProperty(Name);
+            if (prop != null && prop.Type == JTokenType.Array)
```

Comment:
We shouldn't be messing with the default behaviors of the property setter.  This just informs the system how to make the reflection calls.  Seems like the problem is in the ReactMethod itself, not the setter.